Engineering.md

```diff
@@ -1,9 +1,47 @@
 # Engineering Roles
 
+## Junior Engineer
+
+> I am an enthusiastic learner, continually improving my value to an Octopus team.
+
+### I am an egoless, team oriented developer
+
+- Other team members genuinely want me on their team.
+- I happily take responsibility for what I produce.
+- I avoid going dark and I know when to call out for help.
+- I communicate professionally and keep stake holders engaged at all times.
+- I am learning about the processes of the team and do my best to follow them, with the help of others around me.
+- Even when I disagree with something I continue to work enthusiastically, learning from the others in the team.
+- Every venture has mundane tasks, and I work happily and diligently through them.
+- I go out of my way to develop a friendly and professional rapport with my customers and their team members.
+- My contributions to team discussions are net-positive, informative, and helpful.
+
+### I am establishing my ability to write and ship high quality code
+
+- I am learning to write the correct code; code that meets my team's needs.
```

Comment:
the correct code feels ambiguous. I'd go with:
```suggestion
- I am learning to write the code that meets my team's needs.
```

Reply:
hmm, I wonder if there is a better way to say this. Maintainable code?

Reply:
How about something like quality or high quality code.